textarea/textarea.go

```diff
@@ -46,6 +46,8 @@ type KeyMap struct {
 	Paste                   key.Binding
 	WordBackward            key.Binding
 	WordForward             key.Binding
+	MoveToBegin             key.Binding
+	MoveToEnd               key.Binding
```

Comment:
I would probably name this `InputStart` / `InputEnd` to match `LineStart` / `LineEnd`

Reply:
Sure, done.